### PR TITLE
Add Contents.json for all iOS .imageset directories.

### PR DESCRIPTION
**Pull request by sdefresne**
The actool requires the Contents.json file to have required
metadata about the image to compile but some were missing.

Automatically generate all missing Contents.json file via a
shell script assuming that all icons are universal and have
a 1x, 2x and 3x variant.

Fixes issue #630.

### CODE REVIEW

**Comment by sdefresne** [comment]
Note that this change was automated using the following shell command:

  for dir in $(find . -name '*.imageset'); do
      if [[ ! -f $dir/Contents.json ]]; then
          name="${dir##*/ios/}"
          name="${name%%.imageset}"
          cat > $dir/Contents.json <<__END__
  {
       "images": [
           {
               "filename": "${name}.png",
               "idiom": "universal",
               "scale": "1x"
           },
           {
               "filename": "${name}_2x.png",
               "idiom": "universal",
               "scale": "2x"
           },
           {
               "filename": "${name}_3x.png",
               "idiom": "universal",
               "scale": "3x"
           }
       ],
       "info": {
           "author": "xcode",
           "version": 1
       }
  }
  __END__
          git add $x/Contents.json
      fi
  done

**Comment by sdefresne** [comment]
Not sure how to get this reviewed. Mentioning some people that I think work on this project (@liquidx @jestelle) so that they may triage.

File: action/ios/ic_accessible_18pt.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (createyourperonalaccount) [inline action/ios/ic_accessible_18pt.imageset/Contents.json]: move 2017
createyourperonalaccoun

**Comment by liquidx** [comment]
@sdefresne: Were all of them missing the Contents.json, or just some of them?

We should merge this, but @shyndman can you sanity check?

**Comment by sdefresne** [comment]
@liquidx: Only some of the imageset were missing Contents.json.

For example, `action/ios/ic_pan_tool.imageset` was missing Contents.json while `action/ios/ic_settings_ethernet.imageset` included Contents.json.

I noticed this because I tried to use `action/ios/ic_touch_app.imageset` that miss the file. Then I looked at random imageset and noticed that roughly 1 in 7 (1129 out of 7777 imageset) were missing the Contents.json file and wrote the script to create all of them.

**Comment by liquidx** [comment]
Good to know. If it is only missing from a portion, it means what we have to generate those imagesets internally has omitted generating those. So we need to fix that in addition to just adding these files one-off.

**Comment by sdefresne** [comment]
I agree. Do you still want to review this change or do you want to instead fix the generator and push a fixed version? This is kind of blocking Chrome as we'd like to use ic_touch_app but it does not have a Contents.json file.

**Pull request merged**